ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-provision-chain.yaml

```diff
@@ -0,0 +1,30 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-provision
+  steps:
+    - ref: ipi-install-rbac
+    - ref: openshift-cluster-bot-rbac
+    - ref: aws-provision-vpc-disconnected
+    - ref: aws-provision-bastionhost
+    - ref: aws-provision-cco-manual-users-static
```

Comment:
How about renaming as "aws-provision-static-iam-users" (i.e. remove the word "cco-manual")?

Reply:
`cco-manual` means these users are for CCO in manual mode, I think it's more clear.